staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -634,6 +644,29 @@ func (r *crdHandler) getOrCreateServingInfoFor(uid types.UID, name string) (*crd
 		structuralSchemas[v.Name] = s
 	}
 
+	var openAPIModels proto.Models
+	if utilfeature.DefaultFeatureGate.Enabled(features.ServerSideApply) && r.staticOpenAPISpec != nil {
+		specs := []*spec.Swagger{}
+		for _, v := range crd.Spec.Versions {
+			s, err := builder.BuildSwagger(crd, v.Name, builder.Options{V2: false, StripDefaults: true, StripValueValidation: true})
+			if err != nil {
+				utilruntime.HandleError(err)
+				return nil, fmt.Errorf("the server could not properly serve the CR schema")
```

Comment:
I think using errors.Wrapf is better, preserving err.

Reply:
This repeats a pattern from earlier in the function, see commented blocks above. It seems that errors are deliberately suppressed here. e.g., they're not errors the user will gain benefit from seeing. @sttts, is there a reason for that?